### ISSUE TITLE
Is there a good way to reduce web page differences between PC browser and mobile browser?

### ISSUE THREAD

**Issue by Insullone** [necolas/react-native-web#1815]
I am writing a chat web app with Expo ('react-native' alias for 'react-native-web' in code), and it seems very challenging to have expected behavior on both PC and mobile browsers. I want to create a chat page which has messages shown above and an input bar fixed at the bottom. My chat screen/page looks OK on PC browser, but has issues on mobile browser:
(1) When click on Input bar (TextInput), keyboard pops up, leading to messages moving up and off screen.
(2) If messages reach end, new message sent is blocked by input bar. 
I feel it challenging to make pages look good on both browsers, I am very appreciated if any help/hints can be given. Thanks.

My chat code is (simplified):
```javascript
import React, { useState, useEffect, useRef } from 'react';
import { View, Text, FlatList, TextInput, Dimensions } from 'react-native'

const ExperimentalScreen = () => {
  const [messages, setMessages] = useState([]);
  const textInput = useRef(null)
  const flatList = useRef(null)
  const height = useRef(Math.floor(Dimensions.get('window').height)).current;

  useEffect(() => {
    setTimeout(() => {
      flatList.current.scrollToEnd()
    }, 50)
  }, [messages])

  const onSend = (newMsg) => {
    setMessages((prevMsgs) => [...prevMsgs, newMsg])
    textInput.current.clear()
  }

  const renderRow = ({ item }) => {
    return (
      <View>
        <Text style={{fontSize: 40}}>{item.text}</Text>
      </View>
    )
  }

  function genMsg(text) {
    return {
      key: Math.random().toString(36).substring(7),
      text: text,
    }
  }

  return (
    <View style={{flex: 1}}>
      <View style={{height: height - 40, backgroundColor: 'green'}}>
        <FlatList
          ref={flatList}
          data={messages}
          renderItem={renderRow}          
        />
      </View>
      <View style={{position: 'absolute', bottom: 0, left: 0, right: 0, backgroundColor: 'white'}}>
        <TextInput
          ref={textInput}
          multiline
          style={{ height: 40, borderColor: 'gray', borderWidth: 1 }}
          placeholder='Input something...'
          blurOnSubmit
          onSubmitEditing={(e) => onSend(genMsg(e.nativeEvent.text))}
        />
      </View>
    </View>
  )
}
export default ExperimentalScreen;
```

**Comment by necolas** [comment]
Closing as duplicate of #1813. I'm sorry but this issue tracker is not for product UI questions and answers. You might have better luck asking on stackoverflow or other react community forums.
